### DISCUSSION TITLE
Creating waypoint stand alone inserting coordinates

### DISCUSSION THREAD

**Discussion by AndyFag** [osmandapp/OsmAnd#17573] [Q&A]
Hello to everybody,
I have a problem, because I can't create single waypoint by inserting the coordinates.
I would like to do it in my mac desktop app and also in my android device, but:
- In the desktop software I understood that it’s impossible to insert the coordinates: this is a problem, because in the desktop version I normally plan all my long travels
- in the Android version, the process is thought to link a lot of waypoint in a track: but I don’t want always connect the waypoints in a track, for example in my Corsica Map I want to know the location of some water point, but without connect them in a track, because it is a nonsense.
Please, somebody help me to insert coordinates without create a track.
Thanks in advance
Andrea Faggian

**Comment by scaidermern** [comment]
I don't quite understand your issue. You can create individual GPX waypoints without a connection between them. Of course they will belong to a track (or to different tracks).

If you already have coordinates, paste them into the search. The tap on the result to go to this location. Then you can create a new waypoint at these coordinates.

**Comment by AndyFag** [comment]
Hello,Thanks for the answer, but I don’t understand.It is very simple.I want to create for example this waypoint
![IMG_20230708_122628_1CS.jpg](https://github.com/osmandapp/OsmAnd/assets/138881139/9b13c5eb-0b11-4c77-8089-34e41c53e16d)


 And this wp it isn’t part of a track.Well, I need to create a lot of these wp and no one belongs to a track.ThanksAndrea Inviato da iPhoneIl g
iorno 7 lug 2023, alle ore 20:04, Alexander Heinlein ***@***.***> ha scritto:﻿
I don't quite understand your issue. You can create individual GPX waypoints without a connection between them. Of course they will belong to a track (or to different tracks).
If you already have coordinates, paste them into the search. The tap on the result to go to this location. Then you can create a new waypoint at these coordinates.

**Reply by scaidermern** [reply]
Can you explain what the waypoints are for? What do you want to do with them? And why can't they be part of a track?

**Reply by AndyFag** [reply]
In this case the wp are 85 spots in Corsica where It Is possibile to go to swim in the Mountain River.
But in the Sahara (I am a Saharian traveller from 20 years), we normally have a track of also 4000 km, and near the track we have hundred of WP stand alone to indicate water source, a nomadi tent, a Place where there Is an ancien cemetery, and so on.
So, It Is essential to have the possibility to create wp inserting the coordinates, and normally we do this by the desktop and not from the portable device.
I Hope It Is clear.
Tanks
Best
Andrea

**Reply by scaidermern** [reply]
No, not really clear, sorry. I still don't get why you don't want to use a track for this.

Maybe there is some misunderstanding here. A track doesn't need to contain "lines". A track can just contain waypoints. This is perfectly fine.

So start to add your POIs as waypoints to one or more tracks. Either inside OsmAnd. Or create this track at your desktop computer (there are multiple different software available for this task) and then import this track into OsmAnd.

**Reply by pebogufi** [reply]
May be "track" should better be called "gpx file" which can contain 
trk points if it is really a track
or
waypoints (wpt).

**Comment by AndyFag** [comment]
This is a good answer!!! Thanks
Now I understood, because this is the first time that I meet this concept.
But I think that this way is a non sense.
I do an example. I must transform 85 spot of canyoning in Corsica found in a book, in 85 WPs. With Osmand I can do this only from my device Android (I haven't found the way in my desktop app) using marcatori/altro/inserimento coordinate. But if today I have time to insert only 20 spot, I am obliged to close the insertion creating a "track", but if tomorrow I will have time to insert other 30 spot, I can't modify the previous track with the first 20 spot, I am obliged to create a new one with other 30 spots, and so on....
It is useful? I think that this is a non sense, a big non sense.
I used Quo Vadis or Oziexplorer for 15 years and so far, a track is a union of 2 or more WP connected with a line, that I can follow. 
If I must insert the coordinates to create WP, I desire to create them and store them in a place which I choose, being free to move these wp where i want.
I though to have Osmand as my unique software to navigate all over the world, but, so far, is impossible.
Thank in advance for the help and for discussion.
I appreciate it so much.
Andrea Faggian

**Reply by pebogufi** [reply]
@AndyFag 
It is possible to add points to existing file that means to modify the existing "track".
Make a long tap on map to choose the point or  object, then menu...
Azioni / Aggiungo un punti int...
Then select the gpx file.

**Reply by scaidermern** [reply]
Make sure the track you want to add the waypoints to is visible on the map. Then simply create a new waypoint and OsmAnd will ask you whether you want to add it to this existing track.

**Reply by AndyFag** [reply]
Hello,
 thanks to both.
I appreciate very much tour help.
But both the answer don't fix my problem...
The First day I create 30 WPs that I have included in a "track" named Corsica Baignades.
The second day I have time to add other 20 WPs to the "track" called Corsica Baignades..... But I have the coordinates in the book... How can I add the new 20 WPs? Surely no tapping on the screen, and doing this I found another problem: how can I modify the coordinates of this wp? And how can I modify the coordinates of one waypoint that I create some time ago?
Thanks in Advance for your help
Have a nice evening
AF

**Reply by scaidermern** [reply]
> How can I add the new 20 WPs? Surely no tapping on the screen

You enter the coordinates into the search box.

> how can I modify the coordinates of this wp? And how can I modify the coordinates of one waypoint that I create some time ago?

Tap on the waypoint, choose Action / Change marker position. You can't enter new coordinates here but you can simply move the waypoint to a new position. If you really want to add the new position via coordinates then you delete the waypoint and create a new one via the search menu. Alternatively, open the GPX file in a text editor and replace the coordinates.

**Comment by AndyFag** [comment]
Hello,
thanks for your help.
I found as very useful the method to insert the coordinates into the search box! It works perfectly according to my needs.
Thank you very much again.
Andrea
***@***.***




Il giorno 11 lug 2023, alle ore 08:39, Alexander Heinlein ***@***.***> ha scritto:


How can I add the new 20 WPs? Surely no tapping on the screen
You enter the coordinates into the search box.

how can I modify the coordinates of this wp? And how can I modify the coordinates of one waypoint that I create some time ago?
Tap on the waypoint, choose Action / Change marker position. You can't enter new coordinates here but you can simply move the waypoint to a new position. If you really want to add the new position via coordinates then you delete the waypoint and create a new one via the search menu. Alternatively, open the GPX file in a text editor and replace the coordinates.
